script/nonEnglishContent.js

```diff
@@ -0,0 +1,46 @@
+/* eslint-disable no-console */
```

Comment:
I don't think we use camel-case in the names of our script files because they aren't exporting anything - just performing a task.

Can we rename this file to something like `script/lookup-non-english-pages-in-sitemap.js`, or something along those lines? So that another engineer can see just the file name and gather some concept of what it does

Reply:
done!